test/addons/rook-external-cluster/start

```diff
@@ -0,0 +1,176 @@
+#!/usr/bin/env python3
+
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import sys
+import time
+import shlex
+import io
+import tempfile
+import subprocess as subp
+import os.path as ospa
+import logging
+
+import drenv
+from drenv import kubectl
+from drenv import commands
+
+
+# Update this when upgrading rook.
+VERSION = "release-1.10"
+BASE_URL = f"https://raw.githubusercontent.com/rook/rook/{VERSION}/deploy/examples"
+ROOK_CEPH_NAME_SPACE = "rook-ceph"
+
+
+def rook_connect_to_external_storage_cluster(cluster_details, managed_cluster):
+    env = os.environ.copy()
+    script_name = ''
+
+    _, script_name = tempfile.mkstemp(prefix='import-external-cluster-', suffix='.sh')
+
+    download_script_result = subp.run(['curl', '-s', 'https://raw.githubusercontent.com/raghavendra-talur/rook/rtalur-support-context/deploy/examples/import-external-cluster.sh', '-o', script_name], capture_output=True)
+    # download_script_result = subp.run(['curl', '-s', 'https://raw.githubusercontent.com/rook/rook/master/deploy/examples/import-external-cluster.sh', '-o', script_name], capture_output=True)
+    print(f"script {script_name}")
+
+    if download_script_result.returncode != 0:
+        print("Failed to download script: " + download_script_result.stderr)
+        sys.exit(1)
+
+    if not os.path.isfile(script_name):
+        print("Script file does not exist: " + script_name)
+        sys.exit(1)
+
+    with open(script_name) as sf:
+        src = sf.read()
```

Comment:
We copy the script just to read it? we could download to stdout, and read it from the sub process.